### PR TITLE
cleanup: EC2 cleanup

### PR DESCRIPTION
**Pull request by dassana-junix**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,4 +11,4 @@ engine/src/main/resources/content/**
 .idea
 **/target/**
 .aws-sam/**
-content/workflows_unsorted/**
\ No newline at end of file
+content/workflows_unsorted/**
```
Comment (gauravphoenix) [inline .gitignore:14]: where is it used?

**Review by kloading — COMMENTED** [review]
Minor naming fixes. All policies were correctly re-named to the undesirable case.

**Review by kloading — APPROVED** [review]

**Comment by gauravphoenix** [comment]
@dassana-junix - can you address @kloading's comments?

**Comment by kloading** [comment]
> @dassana-junix - can you address @kloading's comments?

@gauravphoenix I committed a fix which addressed my comments since it was very minor. It is ready to be merged

**Comment by parth-dassana** [comment]
There are still quite a few issues. Going to close this out in lieu of one that I did.

**Pull request closed without merging**